docs/docs/deployment.md

```diff
@@ -128,6 +128,40 @@ spec: # v1.PodSpec
   runtimeClassName: contrast-cc
 ```
 
+### Pod resources
+
+Contrast workloads are deployed as one confidential virtual machine (CVM) per pod.
+In order to configure the CVM resources correctly, Contrast workloads require a stricter specification of pod resources compared to standard [Kubernetes resource management].
+
+The CVM memory is calculated from the sum of the memory limits of the containers and a static overhead for daemons.
+InitContainers aren't taken into account.
```

Comment:
From reading these two sentences, it's still not clear to me how this is working. What daemons are we talking about? And what does the initContainer thing mean for me as a user?

Reply:
I tried to explain things in more detail, ptal.